csharp/Svix/SvixClient.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Svix.Abstractions;
+using Svix.Api;
+using Svix.Client;
+using Svix.Models;
+using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Svix
+{
+    public sealed class SvixClient : ISvixClient
+    {
+        private readonly ILogger _logger;
+        
+        private readonly SvixClientOptions _options;
+
+        public SvixClient(SvixClientOptions options, ILogger<SvixClient> logger = null)
```

Comment:
In our other languages we use `options` as our optional variables and pass the token in as its own parameter to the client, I'm wondering if we should do the same here for consistency
 ```suggestion
        public SvixClient(string accessToken, SvixClientOptions options, ILogger<SvixClient> logger = null)
```

Reply:
I'll double check the other clients and to see what you mean. I've tried to create multiple constructors here to support one-off client use, and .Net Core Dependency Injection.